ui/src/main/java/com/hyperwallet/android/ui/view/widget/DateDialogFragment.java

```diff
@@ -0,0 +1,118 @@
+package com.hyperwallet.android.ui.view.widget;
+
+import android.app.DatePickerDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.widget.DatePicker;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.hyperwallet.android.hyperwallet_ui.R;
+
+import java.util.Calendar;
+
+public class DateDialogFragment extends DialogFragment {
+
+    public static final String TAG = DateDialogFragment.class.getName();
+    private static final String ARGUMENT_DATE = "ARGUMENT_DATE";
+    private static final String ARGUMENT_FIELD_NAME = "ARGUMENT_FIELD_NAME";
+    private OnSelectedDateCallback mOnSelectedDateCallback;
+    private DateUtil mDateUtil;
```

Comment:
why not initialize date util as final member?

Reply:
@peter-joseph Done